ptx/sec_deriv_basic_rules.ptx

```diff
@@ -232,7 +232,17 @@
           <caption>A graph of <m>f(x) = x^3</m>, along with its derivative <m>\fp(x) = 3x^2</m> and its tangent line at <m>x=-1</m></caption>
           <!-- START figures/fig_xcubedwithderiv.tex -->
           <image xml:id="img_xcubedwithderiv" width="47%">
-            <description></description>
+            <shortdescription>
+              Graph of function x^3, its derivative and a tangent line drawn at point (-1,-1).
+            </shortdescription>
+            <description>
+              <p>
+                The y-axis is drawn between -4 and 4 and the x-axis is drawn between -2 and 2.
```

Comment:
In the long description, you should put math tags around the variables, even when they're on their own.
It's also not a bad idea to put them around the numbers as well, especially negative numbers, since a hyphen and a minus sign are not the same character. (`-2` is hyphen 2; `<m>-2</m>` is minus two)